demos-and-sample-apps/sdk-browser-overview.md

```diff
@@ -8,46 +8,46 @@ position: 0
 
-The "Telerik UI for Xamarin Samples" is the application you can find in the app stores (Google Play, Apple App Store and  Windows Store). It is highly polished and has many scenario-specific use cases designed to show off a subset of features of each component.
+The **Telerik UI for Xamarin Samples** is the application you can find in the app stores (Google Play, Apple App Store and  Windows Store). It is highly polished and has many scenario-specific use cases designed to show off a subset of features for each component.
 
-The application can be accessed in the following ways:
+You can access the Xamarin Samples application in the following ways:
 
-* It can be found on your computer in the **/[installation-path]/Telerik UI for Xamarin [version]/QSF/** folder;
-* You can explore the code directly in the [Samples Application repository on GitHub](https://github.com/telerik/telerik-xamarin-forms-samples/tree/master/QSF);
+* If you have already installed **Telerik UI for Xamarin**, navigate to the **/[installation-path]/Telerik UI for Xamarin [version]/QSF/** folder and open the `QSF.sln` file;
+* You can explore the code directly in the [Xamarin Samples repository on GitHub](https://github.com/telerik/telerik-xamarin-forms-samples/tree/master/QSF);
 * You can install it from the corresponding app store ([Google Play](https://play.google.com/store/apps/details?id=com.telerik.xamarin&hl=en), [Apple App Store](https://apps.apple.com/dm/app/telerik-ui-for-xamarin-examples/id1083924868) or [Windows Store](https://www.microsoft.com/en-us/p/telerik-ui-for-xamarin-demo/9pld1kn2tcxs)).
 
-The image below shows the Samples application main view with all the Telerik UI for Xamarin controls, there are **Latest** and **Featured** sections to give you quick overview of the features and components recently introduced.
+The image below shows the Xamarin Samples application main view with all the Telerik UI for Xamarin controls. There are the **Latest** and **Featured** sections that give a quick overview of the features and recently introduced components.
 
 ![](images/samplesapplication_0.png)
 
-The next images shows a concrete example from the Samples Application (Calendar &amp; Scheduling Appointment Template example):
+The following images show a concrete example from the Xamarin Samples application (Calendar &amp; Scheduling Appointment Template example):
 
 ![](images/samplesapplication_1.png)
 
 ### Telerik UI for Xamarin Samples Application Dark Mode
 
-The Telerik UI for Xamarin Samples Application has a dark mode support on Android and on iOS. You can change the app theme by clicking on a button. 
+The Telerik UI for Xamarin Samples application has a dark mode support on Android and on iOS. You can change the app theme by clicking on a button. 
 
 ![Telerik Xamarin controls Light Dark appearance](images/dark-light-mode-xamarin-telerik-app.gif)
 
-Here is how the Samples App loos when Dark Mode is applied:
```

Comment:
Isn't this too much? I believe that this is not essential for the article. We already have a gif that shows how dark mode looks.